webservices/docs.py

```diff
@@ -1225,4 +1225,10 @@ def add_ytd(var):
 The Operations log contains details of each report loaded into the database. It is primarily
 used as status check to determine when all of the data processes, from initial entry through
 review are complete.
-'''
\ No newline at end of file
+'''
+LEGAL = '''
+Explore relevant statutes, regulations, Commission actions and court cases.
```

Comment:
Can users see court cases in /legal/?

Reply:
I cribbed it from 
<img width="1125" alt="screen shot 2018-09-18 at 4 11 02 pm" src="https://user-images.githubusercontent.com/2066204/45721601-80ad6500-bb5d-11e8-8ad1-5414c0c90f0b.png">

Should I drop court cases?

Reply:
It looks like court cases aren't in the legal search, so I think so - otherwise people might expect them to be there.

Reply:
Looks like https://www.fec.gov/legal-resources/court-cases/ is a hard-coded or wagtail page, and the "browse" option takes them to classic.fec.gov